src/relayserver/RelayServer.ts

```diff
@@ -949,4 +986,8 @@ export class RelayServer extends EventEmitter {
     }
     return 1000
   }
+
+  address2topic (address: string): string {
```

Comment:
We already have 'event2topic' in the Utils.ts
I believe it will be beneficial to keep these utility functions in the same utility file instead of inside the server

Reply:
Fixed